apps/probate/probate-cron-hmrc-extract/probate-cron-hmrc-extract.yaml

```diff
@@ -14,7 +14,7 @@ spec:
   chart:
     spec:
       chart: probate-cron
-      version: 0.0.17
+      version: 0.0.18
```

Comment:
there's no tag 0.0.18, it's v0.0.18

Reply:
we are referring chart version in container registry(hmctspublic) with is 0.0.18
hmctspublic/probate-cron                          	0.0.18       	0.0.18     	Probae cron jobs